File: core/src/io/cucumber/utils/helper/InputHelper.java
package io.cucumber.utils.helper;

import com.badlogic.gdx.Gdx;

import static io.cucumber.utils.constant.GameConstants.MAX_POINTERS_COUNT;

public class InputHelper {

    public static int getLastTouchX() {
        int x = 0;
        for (int i = 0; i < MAX_POINTERS_COUNT; i++) {
            if (Gdx.input.isTouched(i)) {
                x = Gdx.input.getX(i);
            }
        }
        return x;
    }

    public static int getLastTouchY() {
        int y = 0;
        for (int i = 0; i < MAX_POINTERS_COUNT; i++) {
            if (Gdx.input.isTouched(i)) {
                y = Gdx.input.getY(i);
            }
        }
        return y;
    }

    private InputHelper() {
    }
}

File: core/src/io/cucumber/utils/InputHelper.kt
package io.cucumber.utils

import com.badlogic.gdx.Gdx

object InputHelper {

    const val MAX_POINTERS_COUNT = 20
    const val MIN_FLING_DISTANCE = 100


    fun getLastTouchX(): Int {
        var x = 0
        for (i in 0 until MAX_POINTERS_COUNT) {
            if (Gdx.input.isTouched(i)) {
                x = Gdx.input.getX(i)
            }
        }
        return x
    }

    fun getLastTouchY(): Int {
        var y = 0
        for (i in 0 until MAX_POINTERS_COUNT) {
            if (Gdx.input.isTouched(i)) {
                y = Gdx.input.getY(i)
            }
        }
        return y
    }

}